plugins/rapture/nexus-coreui-plugin/src/main/java/org/sonatype/nexus/coreui/RepositoryComponent.groovy

```diff
@@ -12,14 +12,13 @@
  */
 package org.sonatype.nexus.coreui
 
-import javax.annotation.Nullable
-import javax.inject.Inject
-import javax.inject.Named
-import javax.inject.Singleton
-import javax.validation.Valid
-import javax.validation.constraints.NotNull
-import javax.validation.groups.Default
-
```

Comment:
looks like you also have wrong import order for groovy files:

https://github.com/sonatype/codestyle#imports